assignments/hackyourtemperature/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "hackyourtemperature",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "nodemon server.js",
+    "test": "jest",
+    "format": "prettier --write 'src/**/*.js'" 
```

Right now this command is failing because prettier do not need the `'` between its sources

this should fix it:
```suggestion
    "format": "prettier --write src/**/*.js" 
```

In addition, since most of your file are not under src, you might want to do have this:
```suggestion
    "format": "prettier --write **/*.js" 
```

I will let you try it and see what it updates and how